contexts/SubscriptionContext/tests/Integration/UseCases/AddSubscription/AddSubscriptionUseCaseTest.php

```diff
@@ -139,4 +141,19 @@ public function testGivenDataThatNeedsToBeModerated_requestNotBeMailed() {
 		$useCase->addSubscription( $request );
 	}
 
+	public function testNewMailAddressFromSubscription(): void {
+		$useCase = new AddSubscriptionUseCase( $this->repo, $this->validator, $this->mailer );
+		$class = new ReflectionClass( AddSubscriptionUseCase::class );
```

Comment:
Just to avoid misunderstandings: I was kidding with the sneaking part. If the sentence contains "evil" I'm probably not being serious.

---

As far as I can tell there was no uncovered desired behavior and not even additional behavior of the non-harmful type that was not tested. Just dead code.

Perhaps you missed the part where I said that the used constructor parameter is already tested? Before removing your test I verified this by breaking the code and the failure being caught by both your test and an earlier one.